### PR TITLE
feat(osc): add a user-scoped Eos template, drop the patch variant

### PR DESCRIPTION
**Pull request by michelhonold**
Closes the outstanding item from the bundled-Eos-template fix: *"hardware
verification against a physical Eos console. The axis convention is documented,
not observed. No test can catch a wrong convention."*

It is now observed. Verified against ETCnomad on macOS with the Augment3d demo
show, and end-to-end from a Pi driving four moving lights.

## Templates

| | Address | |
|---|---|---|
| **ETC Eos** | `/eos/chan/[markerid]/xyz` | unchanged |
| **ETC Eos (User 99)** | `/eos/user/99/chan/[markerid]/xyz` | new |
| ~~ETC Eos (Patch)~~ | `/eos/set/patch/[markerid]/augment3d/position` | removed |

The user-scoped variant addresses a specific Eos user instead of whichever one
is current, so a 30 Hz stream cannot disturb the operator's command line. The
patch variant wrote patch data rather than live position, which never reaches
cue data, and the live-focus route already covers the use case.

Removing a template is safe for existing configs: `template_id` is not read at
render time, so a row carrying the old id keeps its own address and args and
goes on working. Only the picker's "which builtin is this from" link goes stale.

## What the hardware said

```
sent   /eos/chan/401/xyz  ,fff  [-5.5, 4.25, 2.75]
Eos    X Focus = -5.5    Y Focus = 4.25    Z Focus = 2.75
```

1:1, in metres, no transposition. Confirmed on a Scenic Element Movable and on a
moving light, so both readings of the address hold.

Two claims died in the process:

- **`Eos rejects the i a bare 0 gives`**, asserted in a code comment. It does
  not: `,fffiii` was accepted and applied correctly. The comment is gone rather
  than reworded; there is nothing non-obvious left to say about a float literal.
- **The website manual documented `[x] [z] [y]`**, the pre-fix swapped order, as
  correct across three pages. Corrected separately in the website repo.

## Verification tooling

`scripts/hw_validation/eos_console_probe.py` drives the deployed
`OscTransmitterManager` and `OscService`, so the bytes on the wire are the app's
rather than a re-render. Its `verify` command reads Eos's own parameter values
back over OSC and asserts the axis mapping, exiting 0/1 with nobody watching
Augment3d:

```sh
poetry run python scripts/hw_validation/eos_console_probe.py \
    --host <EOS_IP> --channel 401 verify
```

`test` / `sweep` / `stream` remain for judging movement by eye.

Three console behaviours are encoded in it, because each one silently imitates a
broken template and each cost real debugging time:

- **Eos discards OSC arriving on a duplicate-subnet interface** with no error and
  no log line, after the kernel has already delivered the packet to its socket.
  An `/eos/ping` control therefore runs before any silence is interpreted. This
  one sent the investigation down a false path for an hour.
- **Eos reports only changes.** Re-sending a held value, or re-selecting an
  already-selected channel, is silent in a way indistinguishable from rejection.
  The probe parks the selection and swaps probe values when the channel already
  holds them.
- **Diagnostics (Tab 99) does not log inbound OSC messages** at all, so an empty
  feed proves nothing.

Mutation-checked rather than assumed: a Y/Z transposition and a wrong address are
both caught, exit 1, with the offending axis named.

## Docs

- `docs/EOS_NOMAD_VERIFICATION.md` - the procedure, written to be re-run: console
  setup, failure modes, the Eos behaviours above, and a dated known-good baseline.
- Help drawer and hw_validation README updated.

## Verification

`make ci-remote` green: 981 passed, 100% coverage, build clean. **It ran locally
on the Mac, not the Pi** - pi66 is the packaged-`.deb` device with no git
checkout, so the gate always falls back. This has no aarch64 coverage.

### CODE REVIEW

**Comment by michelhonold** [comment]
## Review fixes applied (`a59d645`)

Code review found 13 issues, 12 of them in the new probe. All fixed and re-verified against the live console; both templates still PASS 1:1, exit 0.

The four that mattered:

- **`verify` was broken at its own documented defaults.** `--channel` and `--park-channel` both defaulted to `1`, so it parked the selection on the channel it then re-selected. Eos reports only changes, so the baseline came back empty and the operator was told to re-patch a channel that was fine. Park now defaults to `--channel + 1`; an explicit collision is refused.
- **The send counter counted calls, not sends.** `OscService.send` has several early returns and swallows client failures, so a sweep at an unresolvable host printed "600 messages" and exited 0. Now reads `stats_for().total_sent` and surfaces swallowed errors. Verified: that case now reports `FAIL: nothing left the transmitter`, exit 1.
- **An OSC bundle decoded to `("#bundle", [])` and vanished**, which surfaced as "the console is discarding traffic" — the one diagnosis guaranteed to send someone down a rabbit hole.
- **A `T`/`F` argument satisfied the numeric check** (`bool` is an `int` subclass) and could overwrite a real Focus reading, fabricating an axis mismatch.

Plus: SPDX header (this was the only unlicensed source file in the tree), stale docstrings naming the deleted patch template and repeating the int-vs-float claim this PR's own hardware run disproved, `--rate` printing a rate the row wasn't snapped to, unhandled Ctrl-C on a prompt that advertises it, `/eos/subscribe` left on at exit, and a preview test whose assertions couldn't fail.

Two notes on the review itself: its `--port 0` repro for the counting bug wouldn't have shown it (`OscDestinationConfig` coerces an out-of-range port back to 8000, so that send goes somewhere valid) — an unresolvable host is needed. And I did **not** add unit tests for the probe's pure helpers: `scripts/` has no test precedent, none of the eight sibling probes are tested, and it sits outside both `--cov=openfollow` and the lint target.

`make ci-remote` green again: 8722 unit, 981 integration, 100% coverage, build clean. Log line confirms `No testing Pi reachable ... falling back to local`, so this remains Mac-only with no aarch64 coverage.

File: scripts/hw_validation/README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ to grow into a fuller two-Pi validation suite (see the tracking issue).
 | `raw_udp_probe.py` | both | Dependency-free UDP reachability preflight – tells a network drop apart from an app bug. |
 | `psn_packet_size_probe.py` | DUT | Builds real multi-tracker PSN datagrams with the deployed encoder, round-trips them through a loopback socket at 1500 vs 65535, and guards the receiver's `recvfrom` buffer (#463). |
 | `osc_socket_options_probe.py` | DUT | Builds clients via the deployed `OscService._make_client` and asserts the broadcast/multicast socket options (#482). |
+| `eos_console_probe.py` | workstation | Drives the deployed `OscTransmitterManager` at an Eos console / ETCnomad through single-axis sweeps, so the bundled ETC Eos templates' X/Y/Z convention can be observed in Augment3d. Exits `0` (sent) / `1` (nothing transmitted); the axis verdict is the operator's. |
```
[the remark this answers is not in the document]
Comment (michelhonold) [reply scripts/hw_validation/README.md]: Valid, fixed in `78f9d8e`. That row was written before the script grew its `verify` command and never updated. It now distinguishes the two: `verify` reads Eos's parameter values back and asserts the mapping, so its exit code *is* the verdict; `test` / `sweep` / `stream` still report only whether the sends reached the socket.

File: docs/EOS_NOMAD_VERIFICATION.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+but at Eos's processing layer.
+
+### OSC TX (required for `verify`)
+
+Same page. This is what lets the console report back, turning the check from
+"watch Augment3d and judge" into a machine-readable assertion:
+
+- **OSC TX** on.
+- **OSC UDP TX Port** = `8001` (must match `--rx-port`).
+- **OSC UDP TX IP Adresse** = the address of the machine running the probe.
```
[the remark this answers is not in the document]
Comment (michelhonold) [reply docs/EOS_NOMAD_VERIFICATION.md]: Valid, fixed in `78f9d8e`. "Adresse" was transcribed from a German-language console during the original verification run and does not match what an English console shows.

Same root cause a few lines further down, which this comment did not cover: the manual also quoted the German error text `>>Fehler: Dieses Gerat benutzt keine DMX-Adressen`. Replaced with a plain-English description rather than an English string I have not actually observed on screen.
